File: go/wam/5702.go
const WamSearchTheWebFunnel = 5702 // channel: regular

type SearchTheWebFunnelEvent struct {
	MessageType MESSAGETYPE `wam:"message_type"` // field 1
	StwEntryPoint STWENTRYPOINT `wam:"stw_entry_point"` // field 2
	StwFormat STWFORMAT `wam:"stw_format"` // field 3
	StwInteraction STWINTERACTION `wam:"stw_interaction"` // field 4
	ImageSearchFailedErrorType IMAGESEARCHFAILEDERRORTYPE `wam:"image_search_failed_error_type"` // field 5
}

File: rust/wam/5702.rs
pub const WAM_SEARCH_THE_WEB_FUNNEL: u32 = 5702;

#[derive(Debug, Default)]
pub struct SearchTheWebFunnelEvent {
    /// field 1, wire `message_type`
    pub message_type: Option<MESSAGETYPE>,
    /// field 2, wire `stw_entry_point`
    pub stw_entry_point: Option<STWENTRYPOINT>,
    /// field 3, wire `stw_format`
    pub stw_format: Option<STWFORMAT>,
    /// field 4, wire `stw_interaction`
    pub stw_interaction: Option<STWINTERACTION>,
    /// field 5, wire `image_search_failed_error_type`
    pub image_search_failed_error_type: Option<IMAGESEARCHFAILEDERRORTYPE>,
}